packages/runtime/__tests__/load-remote.spec.ts

```diff
@@ -399,6 +417,25 @@ describe('loadRemote with manifest.json', () => {
     Global.__FEDERATION__.__INSTANCES__ = [];
     globalThis.__FEDERATION__.__DEBUG_CONSTRUCTOR__ = undefined;
   });
+
+  it('manifest.json with query', async () => {
+    const FM = new FederationHost({
+      name: '@demo/host',
+      remotes: [
+        {
+          name: '@demo/main',
+          entry:
+            'http://localhost:1111/resources/main/federation-manifest.json?query=2',
```

Comment:
do we have federation manifest open sourced yet? <3  
If not, no problem, but i want that manifest as some point in near future hahaha!
We can discuss after holiday as i have a local branch with VMOK StatsPlugin partially extracted

Reply:
Not yet , but we plan to port more plugins to here next year , includes manifest plugin .

Reply:
No problem :D